src/webui/api/appcontroller.cpp

```diff
@@ -649,8 +649,22 @@ void AppController::setPreferencesAction()
     }
     // Current network interface address
     if (hasKey("current_interface_address")) {
-        const QHostAddress ifaceAddress {it.value().toString().trimmed()};
-        session->setNetworkInterfaceAddress(ifaceAddress.isNull() ? QString {} : ifaceAddress.toString());
+        const QString value = it.value().toString().trimmed();
+        bool ok = false;
+        uint option = value.toUInt(&ok);
+
+        if (ok) {
+            if (option == 1)
```

Comment:
for 3-way or more cases better use a `switch()`.

Reply:
I wanted to avoid the extra space taken by the `break` statement on each branch.